rpc/documentation/api.md

```diff
@@ -635,12 +639,53 @@ search for transactions involving the private key's associated payment address.
 
 **Expected errors:**
 
+- `InvalidArgument`: The private passphrase is incorrect.
+
 - `InvalidArgument`: The private key WIF string is not a valid WIF encoding.
 
+- `InvalidArgument`: A rescan height was specified, but the rescan option was 
+  not set.
+
+- `InvalidArgument`: A negative rescan height was passed.
+
 - `Aborted`: The wallet database is closed.
 
+- `NotFound`: The account does not exist.
+
+**Stability:** Unstable
+
+___
+
+#### `ImportScript`
+
+The `ImportScript` method imports a script into the wallet.  A rescan may 
+optionally be started to search for transactions involving the script, either 
+as an output or in a P2SH input.
+
+**Request:** `ImportScriptRequest`
```

Comment:
I'll add scan_from, but I don't think we should have accounts for scripts as scripts can span no accounts (OP_TRUE) or many accounts (2-of-2 P2SH involving 2 accounts from the same wallet). That's why wallet also puts them in txstore unencrypted now, and in the future I would consider stripping them from the address manager completely because they're fundamentally different from an account or private key.